lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -30,66 +28,121 @@ class MinitestAssertions < Base
           RESTRICT_ON_SEND = %i[
             assert_equal
             assert_not_equal
-            refute_equal
             assert_nil
             assert_not_nil
+            assert_empty
+            assert_not_empty
+            refute_equal
             refute_nil
+            refute_empty
           ].freeze
 
-          # @!method minitest_equal_assertion(node)
-          def_node_matcher :minitest_equal_assertion, <<~PATTERN
+          # @!method minitest_equal(node)
+          def_node_matcher :minitest_equal, <<~PATTERN
             (send nil? {:assert_equal :assert_not_equal :refute_equal} $_ $_ $_?)
           PATTERN
 
-          # @!method minitest_nil_assertion(node)
-          def_node_matcher :minitest_nil_assertion, <<~PATTERN
+          # @!method minitest_nil(node)
+          def_node_matcher :minitest_nil, <<~PATTERN
             (send nil? {:assert_nil :assert_not_nil :refute_nil} $_ $_?)
           PATTERN
 
+          # @!method minitest_empty(node)
+          def_node_matcher :minitest_empty, <<~PATTERN
+            (send nil? {:assert_empty :assert_not_empty :refute_empty} $_ $_?)
+          PATTERN
+
           def on_send(node)
-            minitest_equal_assertion(node) do |expected, actual, fail_message|
-              prefer = replace_equal_assertion(node, expected, actual,
-                                               fail_message.first)
-              add_an_offense(node, prefer)
+            minitest_equal(node) do |exp, act, msg|
+              on_assertion(node, EqualAssertion.new(node, exp, act, msg.first))
+            end
+
+            minitest_nil(node) do |act, msg|
+              on_assertion(node, NilAssertion.new(node, act, msg.first))
             end
 
-            minitest_nil_assertion(node) do |actual, fail_message|
-              prefer = replace_nil_assertion(node, actual,
-                                             fail_message.first)
-              add_an_offense(node, prefer)
+            minitest_empty(node) do |act, msg|
+              on_assertion(node, EmptyAssertion.new(node, act, msg.first))
             end
           end
 
-          private
+          def on_assertion(node, assertion)
+            preferred = assertion.replaced
+            add_offense(node, message: message(preferred)) do |corrector|
+              corrector.replace(node, preferred)
+            end
+          end
 
-          def add_an_offense(node, prefer)
-            add_offense(node, message: message(prefer)) do |corrector|
-              corrector.replace(node, prefer)
+          def message(preferred)
+            format(MSG, prefer: preferred)
+          end
+
+          # :nodoc:
```

Comment:
Is this needed?

Reply:
Yes `Style/Documentation` says Missing top-level documentation comment for class.

Reply:
I mean, is a base class necessary when there’s a single method that is not implemented?

Reply:
Ah I see. I'm sorry I misunderstood.